src/drive/web/modules/move/MoveModal.jsx

```diff
@@ -75,7 +77,9 @@ class MoveModal extends React.Component {
 
     try {
       await Promise.all(
-        entries.map(entry => this.moveEntry(entry._id, entry.dir_id))
+        entries.map(
+          async entry => await this.moveEntry(entry._id, entry.dir_id)
```

Comment:
don't need to add async / await here, right? Since they are declared in the method?

Reply:
No need to `await` 

We have an `async` map, we return `moveEntry` which is a `promise`. So `awaiting` `promise.all` is enough. Nop?